src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -196,6 +197,8 @@ data class IndividualUser(
 
   override fun canCreateDeviceManager() = isSuperAdmin()
 
+  override fun canCreateDocument() = true
```

Comment:
Document creation isn't going to be end-user-accessible at first, right? Should these document permissions require TF Expert instead of being open to everyone?

Or is this just to get the tests passing initially without having to muck around with permissions?

Reply:
This will be addressed in another PR into the feature branch, for now it is captured as a TODO in the main feature branch PR description.